### PR TITLE
Fix TestRegisterSEESubscriptionReconnectsStreamOnError

### LINKED ISSUE #296: TestRegisterSEESubscriptionReconnectsStreamOnError fails on windows
```
--- FAIL: TestRegisterSEESubscriptionReconnectsStreamOnError (0.50s)
        Error Trace:    subscription_test.go:429
        Error:          did not receive event in time
```

I am able to reproduce it only on Windows (tested golang 1.8.3 and 1.6.2, Windows 10 Pro Creators Update).  Can be solved by setting http client timeout to e.g. 50 milliseconds -  PR ok?

### PR DESCRIPTION
**Pull request by ondrej-smola**
fixes #296

### CODE REVIEW

**Comment by ondrej-smola** [comment]
Test fails only for go 1.5 - race detector. Related to https://github.com/golang/go/issues/12262 ?

**Comment by timoreimann** [comment]
@ondrej-smola looks like it's that bug, yeah.

1.5 is ancient old in Go years, we can probably deprecate support for it.

In the meantime, let me try to kick Travis...

**Comment by ondrej-smola** [comment]
waiting for #297

**Comment by timoreimann** [comment]
@ondrej-smola dependent PR has been merged. Could you update this PR?

**Comment by ondrej-smola** [comment]
Sorry, too much going on this week - will update it on weekend/next week.

File: subscription_test.go [hunk 1]
```diff
@@ -20,6 +20,9 @@ import (
 	"testing"
 	"time"
```
Comment (timoreimann) [inline subscription_test.go]: No blank line between stdlib imports please.

File: subscription_test.go [hunk 2]
```diff
@@ -20,6 +20,9 @@ import (
...
 
+	"net"
```
Comment (timoreimann) [inline subscription_test.go]: Please move between line 19 and 20 to respect alphabetical order.

Comment (ondrej-smola) [inline subscription_test.go]: There should be some better way how to check format automatically (`make format` does not help in this case) at least using goimports (they at least sort imports) - as this imports fiddling is very error prone

Comment (timoreimann) [inline subscription_test.go]: Totally agree. Patches welcome. :-)

**Pull request merged**